adding graph desc + wording update (#5257)
DataDog/documentation: content/en/tracing/visualization/resource.md

    * The **Total amount of errors**
    * The amount of **Errors per second**
    * The **% Error Rate**
* Sub-Services: When there are multiple services involved, a fourth graph is available that breaks down your **Total time spent**/**%of time spent**/**Avg time per request** of your service by *services* or *type*.

>     It represents the total/relative/average time spend by traces from the current service into other *services* or *type*.

Reviewer: ```suggestion
    This represents the total/relative/average time spent by traces from the current service to the other *services* or *type*.
```

>     **Note**: For services like *Postgres* or *Redis*, which are "final" operations not calling other service underneath, there is no Sub-services graph.

Reviewer: ```suggestion
    **Note**: For services like *Postgres* or *Redis*, which are "final" operations that do not call other services, there is no sub-services graph.
```
